datafusion/physical-plan/src/joins/hash_join.rs

```diff
@@ -3844,6 +3851,103 @@ mod tests {
         Ok(())
     }
 
+    pub fn build_table_struct(
+        struct_name: &str,
+        a: (&str, &Vec<Option<i32>>),
```

Comment:
can we have a name more meaningful than `a`?
Would be also nice to document the tuple.
if its pub function please add the method description for the users

Reply:
Made it private, was just me being sloppy that made it pub.